src/BPMDisplay.cpp

```diff
@@ -120,8 +120,8 @@ void BPMDisplay::SetBPMRange( const DisplayBpms &bpms )
 		int MaxBPM = INT_MIN;
 		for( unsigned i = 0; i < BPMS.size(); ++i )
 		{
-			MinBPM = std::min( MinBPM, static_cast<int>(std::lrint(BPMS[i])) );
```

Comment:
This is technically not redundant, but we can change to `static_cast<int>(BPMS[i] + 0.5);`

Reply:
I don't really understand what do we optimize here for while giving up readability. Perhaps we should just let the compiler inline it when `-fno-math-errno` since we never check for `errno` anyway? 

https://godbolt.org/z/oE7371jMc

Reply:
I don't think the `lrint`'s affect sync at all, to be clear. I want to change them where possible because msvc (what I'm using to build these on Windows) handles them very badly compared to other compilers. To be honest, if the release builds for Windows are built with gcc, it probably doesn't matter.

It's a considerable savings on msvc , while it barely makes a difference with gcc: https://godbolt.org/z/53xWE949x